### PR TITLE
Hotfixes the reverse revolver appearing in surplus crates

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
del: The Syndicate will no longer prank their operatives by including reverse revolvers in surplus crates.
/:cl:

F E A T U R E

### CODE REVIEW

**Comment by anconfuzedrock** [comment]
this is actually the funniest bug I can fucking imagine

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -1165,7 +1165,8 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 // Role-specific items
 /datum/uplink_item/role_restricted
 	category = "Role-Restricted"
-	exclude_modes = list(/datum/game_mode/nuclear)
+	exclude_modes = list(/datum/game_mode/nuclear
```
Comment (PKPenguin321) [inline code/modules/uplink/uplink_item.dm]: )

**Comment by Emilc4h** [comment]
Honestly, this is kinda hilarious, I wish it'd stay in.

**Comment by lzimann** [comment]
Maybe add a way to traitors see it's a tampered revolver instead of removing them from surplus.

**Comment by Armhulen** [comment]
PLEASE KEEP THIS IN

**Comment by Xhuis** [comment]
It's hilarious but there's no indication of it and I don't want to randomly dick over a traitor's round by having their gun misfire at the wrong time.

**Comment by InsaneHyena** [comment]
FUN REMOVAL

**Comment by anconfuzedrock** [comment]
I mean tbh if it was just the reverse revolver having a lower chance thatd be better but this is better than nothing

**Comment by coiax** [comment]
@Xhuis give the clown revolver an additional examine message for traitors, I guess. Or, when it comes in the syndicate surplus crate, make it hot pink.

**Comment by Xhuis** [comment]
Ehh, maybe, but not for this PR.

**Comment by Robustin** [comment]
2 syndie crates ordered today, 2 fucking clown revolvers inside, 2 times I shot myself in the face with a .357. 

Only thing that salvaged the 2nd round was someone immediately picking up the revolver and shooting themselves in the face as well. 

But this should be common sense, job-specific traitor gear loses all meaning if it comes freely in crates. As coders add more job items it will become very common to get several of them in a crate.

**Comment by PKPenguin321** [comment]
Holy shit robustin that's funny as fuck, I actually literally laughed out loud

A shame this has to go

**Comment by Xhuis** [comment]
S P E E D M E R G E

**Comment by BeeSting12** [comment]
I got two clown revolvers in one Syndicate crate. TWO CLOWN REVOLVERS. I tested each one and shot myself each time. Luckily I had the foresight to bring bruise packs. Later on the HoS shot himself with one of them so I guess it was worth it.

**Comment by Armhulen** [comment]
i got one real one fake so I shot down the hallway with one and then threw the other

yeah it got a few people

but needless to say, REMOVE THIS GETTING SHOT IN THE HEAD SUCKS

**Pull request merged**